File: app-src/src/bin/client_gen/java/Debug.java
import java.io.IOException;
import java.io.OutputStream;

public class Debug {
    private OutputStream stream;

    public Debug(OutputStream stream) {
        this.stream = stream;
    }

    public void draw(model.CustomData customData) {
        try {
            new model.PlayerMessageGame.CustomDataMessage(customData).writeTo(stream);
            stream.flush();
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }
}

File: app-src/src/bin/client_gen/kotlin/Debug.kt
import java.io.IOException
import java.io.OutputStream

class Debug(private val stream: OutputStream) {

    fun draw(customData: model.CustomData) {
        try {
            model.PlayerMessageGame.CustomDataMessage(customData).writeTo(stream)
            stream.flush()
        } catch (e: IOException) {
            throw RuntimeException(e)
        }
    }
}
